src/sidebar/services/frame-sync.js

```diff
@@ -366,9 +366,23 @@ export class FrameSyncService {
 
     guestRPC.on(
       'showAnnotations',
-      /** @param {string[]} tags */ tags => {
-        this._store.selectAnnotations(this._store.findIDsForTags(tags));
+      /**
+       * @param {string[]} tags
+       * @param {boolean} focusFirstInSelection - Whether to give keyboard focus
+       *   to the first annotation in the selection.
+       */
+      (tags, focusFirstInSelection = false) => {
+        const ids = this._store.findIDsForTags(tags);
+        this._store.selectAnnotations(ids);
         this._store.selectTab('annotation');
+
+        // Attempt to transfer keyboard focus to the first selected annotation.
```

Comment:
There could be some confusion about `window.focus` (which actually happens here) vs. focusing the first selected annotation (which happens as a side effect of `setAnnotationFocusRequest`).

Reply:
I've added some comments here to hopefully clarify things.